liquid/stream.py

```diff
@@ -99,31 +105,37 @@ def next(self):
 		@returns:
 			str: the next character
 		"""
+		self._last = self.cursor
 		ret = self.stream.read(1)
 		return ret
 
-	def back(self):
-		"""
-		Put cursor 1-character back
-		"""
-		self.stream.seek(self.cursor - 1)
-
 	def rewind(self):
 		"""
 		Rewind the stream
 		"""
 		self.stream.seek(0)
 
+	def seek(self, pos):
+		"""
+		Seek to a particular position in the stream
+
+		:param pos: The position in the stream to seek to
+
+		:return: The new position in the stream
+		"""
+		return self.stream.seek(pos)
+
```

Comment:
Have you used this function somewhere or we just use `self.stream.seek` everywhere?
If so, we don't have to define this function, right?

Reply:
Yeah, it's not necessary.  I can just use the stream directly without adding the public method.  I'll remove.

Reply:
Wired that codacy didn't report the unused code.

Also see my comments below.

Reply:
Oh, the function was being *used*, but only inside of this class. That's probably why.